### PR TITLE
/set dietime [on|off] command

### PR DESCRIPTION
**Pull request by ConradWeiser**
A toggle which timestamps death messages.

Screenshot

### CODE REVIEW

File: Plugins/Public/playercntl_plugin/Message.cpp [hunk 1]
```diff
@@ -1241,6 +1245,34 @@ namespace Message
...
+		wstring wscParam1 = ToLower(GetParam(wscParam, ' ', 0));
+		bool bShowChatDieTime = false;
+		if (!wscParam1.compare(L"on"))
+			bShowChatDieTime = true;
+		else if (!wscParam1.compare(L"off"))
+			bShowChatDieTime = false;
+		else
+		{
+			PrintUserCmdText(iClientID, L"ERR Invalid parameters");
+			PrintUserCmdText(iClientID, usage);
```
Comment (dsyalex) [inline Plugins/Public/playercntl_plugin/Message.cpp]: return?

File: Plugins/Public/playercntl_plugin/Message.cpp [hunk 2]
```diff
@@ -1241,6 +1245,34 @@ namespace Message
...
+		}
+
+		wstring wscCharname = (const wchar_t*)Players.GetActiveCharacterName(iClientID);
+
+		HookExt::IniSetB(iClientID, "msg.chat_dietime", bShowChatDieTime);
+
+		// Update the client cache.
+		map<uint, INFO>::iterator iter = mapInfo.find(iClientID);
+		if (iter != mapInfo.end())
+			iter->second.bShowChatDieTime = bShowChatDieTime;
```
Comment (dsyalex) [inline Plugins/Public/playercntl_plugin/Message.cpp]: Please can we just:
if (mapInfo.find(iClientID) != mapInfo.end())
    mapInfo[iClientID].bShowChatDieTime = bShowChatDieTime;
?

Comment (ConradWeiser) [reply Plugins/Public/playercntl_plugin/Message.cpp]: I've always been told that when working on a project which other people have started, try to use the same style of operations which the others have been using.

Reference line 1239
```// Update the client cache.
map<uint,INFO>::iterator iter=mapInfo.find(iClientID);
if (iter != mapInfo.end())
	iter->second.bShowChatTime = bShowChatTime;```

This was used in the UserCmd_SetChatTime method.
I'm simply mirroring what it used to complete pretty much the same task.

File: Plugins/Public/playercntl_plugin/Message.cpp (hunk 1 above)
Comment (ConradWeiser) [reply Plugins/Public/playercntl_plugin/Message.cpp]: Whoops. I suppose we don't want to turn it off if invalid parameters were used. 
Fixed.

File: Plugins/Public/playercntl_plugin/Message.cpp (hunk 2 above)
Comment (dsyalex) [reply Plugins/Public/playercntl_plugin/Message.cpp]: Good answer.

**Pull request merged**